scripts/toml_to_moha.py

```diff
@@ -0,0 +1,184 @@
+import tomllib
+import numpy as np
+from os.path import exists
+from os.path import join
+from os import makedirs
+import sys
+sys.path.insert(0, '../')
+import moha
+
+def set_defaults(input_data):
+    '''
+    Function for setting defaults in the parameter dictionary
+    loaded from toml input file.
+
+    Parameters
+        ----------
+        input_data: dict
+            dict loaded from toml input file.
+
+    Returns
+        -------
+        None
+    '''
+
+    # require that defaults.toml exists
+    required_default_paramfile = "defaults.toml"
+    if not exists(required_default_paramfile):
+        raise Exception("Default input file 'defaults.toml' is required.")
+    
+    # load defaults.toml data into default_data
+    default_data = tomllib.load(
+                       open(required_default_paramfile, "rb")
+            )
+
+    # set defaults in input_data
+    for param_type in default_data.keys():
+        # set required param type keys if not specified in input_data
+        if not param_type in input_data.keys():
+            input_data[param_type] = {}
+
+        # set required default param values if not specified in input_data
+        for param in default_data[param_type].keys():
+            if not param in input_data[param_type]:
+                input_data[param_type][param] = default_data[param_type][param]
+            # make all strings lowercase for case-insensitive comparisons
+            data_value = input_data[param_type][param]
+            if type(data_value) == str:
+                input_data[param_type][param] = data_value.lower()
+
+            # set Carbon params as default in Huckel
+            if param_type == "model" and param_type == "huckel":
+                pass
```

Comment:
Add defaults for the Huckel

Reply:
defaults specific to Huckel are now set in the set_defaults function